test/sqllogictest/catalog_raw.slt

```diff
@@ -0,0 +1,21 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+mode cockroach
+
+statement ok
+CREATE TABLE foo (a INT);
+
+simple conn=mz_system,user=mz_system
+SELECT data->'value'->>'name' AS name
+FROM mz_internal.mz_catalog_raw
+WHERE data->>'kind' = 'Item' AND data->'key'->'gid'->'value'->'User' IS NOT NULL;
+----
+foo
+COMPLETE 1
```

Comment:
How does this work without an `AS OF` or without switching to `SERIALIZABLE`?

Reply:
My guess is this bit to mark it as being not in Timeline::EpochMillis https://github.com/MaterializeInc/materialize/pull/24138/files#diff-aaa12f17e3b6a75cb7184863212b12712a8b1f808df572f69dbd8c39b518d27dR723-R739

Reply:
Wow, I guess I'm just surprised that the non-Epoch millis timeline machinery actually works out of the box.